core/src/test/java/com/sequenceiq/cloudbreak/websocket/UptimeNotifierTest.java

```diff
@@ -0,0 +1,43 @@
+package com.sequenceiq.cloudbreak.websocket;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.sequenceiq.cloudbreak.TestUtil;
+import com.sequenceiq.cloudbreak.repository.ClusterRepository;
+import com.sequenceiq.cloudbreak.repository.StackRepository;
+import com.sequenceiq.cloudbreak.service.notification.Notification;
+import com.sequenceiq.cloudbreak.service.notification.NotificationSender;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UptimeNotifierTest extends TestUtil {
+
+    @InjectMocks
+    private UptimeNotifier underTest;
+
+    @Mock
+    private ClusterRepository clusterRepository;
+
+    @Mock
+    private StackRepository stackRepository;
+
+    @Mock
+    private NotificationSender notificationSender;
+
+
+    @Test
+    public void test() {
+        when(clusterRepository.findAll()).thenReturn(generateCluster(3));
+        when(stackRepository.findStackForCluster(anyLong())).thenReturn(stack());
+        doNothing().when(notificationSender).send(any(Notification.class));
+        underTest.sendUptime();
+    }
```

Comment:
Some test cases could be added here, that covers the logic of this class: 
using something like this 
http://docs.mockito.googlecode.com/hg/index.html?org/mockito/ArgumentCaptor.html 
the argument of the notificationSender could be captured; nice assertion can be made on that after ...